From 9a6af2ca74b65ec933d24cc75d42559c19b2de69 Mon Sep 17 00:00:00 2001
From: Hiroki Negishi <hiroki.negishi.bx@renesas.com>
Date: Mon, 13 Nov 2017 11:46:15 +0900
Subject: [PATCH 09/23] Revert "Revert "arm64: dts: r8a7795-es1: Add MFIS
 device node""

This reverts commit a428110d14bb ("Revert "arm64: dts: r8a7795-es1:
Add MFIS device node"")

Signed-off-by: Hiroki Negishi <hiroki.negishi.bx@renesas.com>
---
 arch/arm64/boot/dts/renesas/r8a7795-es1.dtsi | 10 +++++++++-
 1 file changed, 9 insertions(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/renesas/r8a7795-es1.dtsi b/arch/arm64/boot/dts/renesas/r8a7795-es1.dtsi
index 30f72bc..0a6bab8 100644
--- a/arch/arm64/boot/dts/renesas/r8a7795-es1.dtsi
+++ b/arch/arm64/boot/dts/renesas/r8a7795-es1.dtsi
@@ -1,7 +1,7 @@
 /*
  * Device Tree Source for the r8a7795 ES1.x SoC
  *
- * Copyright (C) 2016 Renesas Electronics Corp.
+ * Copyright (C) 2016-2017 Renesas Electronics Corp.
  *
  * This file is based on the arch/arm64/boot/dts/renesas/r8a7795.dtsi
  *
@@ -877,6 +877,14 @@
 			       <&ipmmu_mp1 30>, <&ipmmu_mp1 31>;
 		};
 
+		mfis: mfis@e6260000 {
+			compatible = "renesas,mfis-r8a7795", "renesas,mfis";
+			reg = <0 0xe6260000 0 0x0200>;
+			interrupts = <GIC_SPI 180 IRQ_TYPE_LEVEL_HIGH>;
+			interrupt-names = "eicr0";
+			status = "okay";
+		};
+
 		pfc: pfc@e6060000 {
 			compatible = "renesas,pfc-r8a7795";
 			reg = <0 0xe6060000 0 0x50c>;
-- 
1.9.1

